service-jehlomat/src/main/kotlin/services/DatabaseService.kt

```diff
@@ -227,17 +231,30 @@ class DatabaseServiceImpl(
         }
     }
 
-    override fun insertSyringe(syringe: Syringe): Int {
-        return databaseInstance.insertAndGenerateKey(SyringeTable) {
-            set(it.timestamp, syringe.timestamp)
-            set(it.userId, syringe.userId)
-            set(it.photo, syringe.photo)
-            set(it.count, syringe.count)
-            set(it.note, syringe.note)
-            set(it.demolisherType, syringe.demolisher.name)
-            set(it.gpsCoordinates, syringe.gps_coordinates)
-            set(it.demolished, syringe.demolished)
-        } as Int
+    override fun insertSyringe(syringe: Syringe): String? {
+        for (i in 1 .. numberOfInsertSyringeTries) {
```

Comment:
I am not sure. Constants are not in ABC_DEF format in kotlin ?

Reply:
fixed